service/src/main/resources/api/openapi.yml

```diff
@@ -496,6 +522,62 @@ components:
         failures:
           $ref: '#/components/schemas/FailureMessage'
 
+    SubmitWorkflowRequestBody:
+      type: object
+      properties:
+        workflowGcsUri:
```

Comment:
This differs from the original Cromwell endpoint. 

Instead of taking in a workflow source file, we'll take in a GCS URI.

`workflowUrl` below remains unchanged.